osprey-method-handler.js

```diff
@@ -425,7 +425,13 @@ function jsonBodyHandler (body, path, method, options) {
  */
 function jsonBodyValidationHandler (schema, path, method, options) {
   var jsonSchemaCompatibility = require('json-schema-compatibility')
-  var isRAMLType = schema.constructor === {}.constructor
+  if (Array.isArray(schema)) {
+    schema = schema[0]
+  }
+  if (schema.type === 'json' && typeof schema.content === 'string') {
```

Comment:
what happens if `schema` and no `type` or no `content` property, can't that happen?

Reply:
well it just won't execute the conditional